src/App.js

```diff
@@ -19,6 +20,7 @@ function App() {
         <Route path="/signup" element={<Signup />} />
         <Route path="/trash" element={<Trash />} />
```

Comment:
you may use the `Route` as per the private and public access.

Reply:
Okay will do it.